instrumentation/github.com/gorilla/mux/otelmux/go.mod

```diff
@@ -8,14 +8,21 @@ require (
 	github.com/stretchr/testify v1.10.0
 	go.opentelemetry.io/otel v1.35.0
 	go.opentelemetry.io/otel/metric v1.35.0
+	go.opentelemetry.io/otel/sdk v1.35.0
 	go.opentelemetry.io/otel/trace v1.35.0
 )
 
+require (
```

Comment:
Did Go generate this change, or did you manually update the file?
It's a bit weird to have a new section for indirect dependencies, and a direct one within indirects.

You should generate `go.mod` with `go mod tidy`, not need to update it manually.